docs/examples/embeddings/voyageai.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>It should be Voyageai embeddings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>